src/routes/external-extra/components/ExternalExtra.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import * as React from 'react'
+import { WebView } from 'react-native-webview'
+import { Text } from 'react-native'
+import { stringify } from 'query-string'
+
+export type PropsType = {|
+  url: ?string,
+  postData: ?Map<string, string>
+|}
+
+class ExternalExtra extends React.Component<PropsType> {
+  render () {
+    const {url, postData} = this.props
+    // $FlowFixMe Object.fromEntries is available in Flow version >= 0.99.x: https://github.com/facebook/flow/commit/f92231cf15022c729226b4d440435b3366f9346c
+    const body = !postData ? '' : stringify(Object.fromEntries(postData))
+    return <WebView
+      source={{
+        uri: url,
+        method: postData ? 'POST' : 'GET',
```

Comment:
You said on Friday that it did not work because of specific Headers. Was that the case or was there an other problem?

Reply:
Well the problem was that I sent json in the body together with a `application/x-www-form-urlencoded` Content-Type header. I tried to adjust the Content-Type header, but this is not possible together with a POST request in Android (wtf). So I decided to use `stringify` from `query-string` instead of a JSON body.
This works on Android but not on iOS (wtf). So this PR is still WIP...

Reply:
Whoops, it *does* work on iOS 🤔 🎉  (maybe because of replacing `Object.fromEntries`).

Reply:
Okay, now it doesn't work again iOS (wtf)... I'll have to check this once again. Not sure how to debug this better...

Reply:
Now it works with the added `Content-Type` header 🎉  (Proof:
![image](https://user-images.githubusercontent.com/7025199/58383949-78ef6b80-7fdc-11e9-838e-484460a13a53.png)
)

Reply:
Many wtfs :D